apiserver/facades/client/modelmanager/listmodelsummaries_test.go

```diff
@@ -60,8 +65,13 @@ func (s *ListModelsWithInfoSuite) SetUpTest(c *gc.C) {
 	}
 
 	s.callContext = context.NewEmptyCloudCallContext()
+
+	ctrl := gomock.NewController(c)
```

Comment:
You can't do this in `SetUpTest`, because the `*gc.C` is different to the one in the test methods, and can result in obscure failures.

This is why you will see `setUpMocks` methods everywhere that pass the test's `C` reference to the set up.

Reply:
oh I see, indeed I added the `setUpMock()` func on the tests, but actually I only needed it in `modelmanager_test.go` so finally I removed that mock.